### ISSUE TITLE
RDP Connection Bar is missing

### ISSUE THREAD

**Issue by Frankstar** [asbru-cm/asbru-cm#137]
**Describe the bug**
RDP Connection Bar is missing / vanished with the last update.
Im using xfreerdp - worked like a charm the last months.

**Desktop (please complete the following information):**
 - OS: Fedora 29 
 - Version (asbru) 5.1.0 (asbru-cm-5.1.0~1550940827-1.fc28.noarch)

**Comment by gfrenoy** [comment]
Cannot remember any [recent change](https://github.com/asbru-cm/asbru-cm/commits/master) that would affect xfreerdp behaviour ...

Was xfreerdp upgraded recently ?

Can you also give a bit more details about the issue:

- A screenshot ?
- What version of xfreerdp are you using ?
- What option do you have selected in your settings (fullscreen, embed, fixed widht/height) ?

**Comment by Frankstar** [comment]
im not sure if a screenshot would help ... the connection bar is missing.  (the blue bar in (normaly) middle, Top. )
Im using "Fullscreen".   
But the connection bar is also missing in TAB or Width x Height.   

```
xfreerdp --version
This is FreeRDP version 2.0.0-dev5 (n/a)
```

BUT  - you are right, its not an asbru bug ...
i just connected directly via xfreerdp to the servers and the connectionbar is missing ...
Didnt thought about that earlier.

so it must be something about recent xfreerdp changes.
(or something im not aware of)

**Comment by gfrenoy** [comment]
Thanks for your feedback !

Let us know if there is any new option in xfreedrdp that we should enable...?

**Comment by Frankstar** [comment]
i dont know ... its still missing and i dont know why

**Comment by Frankstar** [comment]
i still have this error.
found this on the remmina Wiki:
```
When I'm in fullscreen mode, I cannot see the "floating toolbar" (the bar at the top of the screen), or I can see part of it but it does not scroll down or receive my input.
A: You are using Gnome Shell and an older version of remmina. This has been fixed under remmina 1.2.0-rcgit.3 compiled with GTK+ 3.10 or newer. Please restart your system after upgrading remmina.
